neuefische.html

```diff
@@ -27,28 +27,36 @@ <h2 id="kontaktformular-heading">Der Weg <span>in deine Digitale Zukunft.</span>
                         name="salutation" 
                         required>
                     <label for="salutationMale">Mann</label>
-                </div>
-                <label for="firstName">Vorname*</label>
-                <input 
-                    id="firstName" 
-                    type="text" 
-                    placeholder="z. B. Jaqueline oder Klaus-Jürgen" 
-                    required><br>
-                <label for="lastName">Nachname*</label>
-                <input 
-                    id="lastName" 
-                    type="text" 
-                    placeholder="z. B. Müller, Meier oder Schmidt" 
-                    required><br>
-                <label for="email">E-Mail-Adresse*</label>
-                <input 
-                    id="email" 
-                    type="email" 
-                    placeholder="z. B. honey2000@gmx-net.de" 
-                    required>
-                
-                <div>    
-                    <span>Ich interessiere mich für ...*</span>            
+                </fieldset>
+                <fieldset>
+                    <legend>Vorname*</legend>
+                    <label for="firstName"></label>
```

Comment:
Here you don't need a fieldset with legend. This would be correct:
```suggestion
                    <label for="firstName">Vorname*</label>
```